esmvalcore/preprocessor/_time.py

```diff
@@ -268,7 +289,6 @@ def clip_timerange(cube: Cube, timerange: str) -> Cube:
     ------
     ValueError
         Time ranges are outside the cube's time limits.
-
```

Comment:
I wonder where removed empty lines come from. Since all our code base has been reformatted with ruff format now, there should be no formatting changes anymore. @bouweandela any idea?

Reply:
They came from the pydocstyle pre-commit hook. I removed that from our pre-commit hooks in #2524, so it should not happen again.

Reply:
I'll see if I can add the ruff equivalent

Reply:
Sorry, didn't quite get (still recovering from a cold), should I correct something or run something again through `pre-commit` or `ruff`?

Reply:
No, it's fine. Just another potential improvement we could make to our automatic formatting.